### DISCUSSION TITLE
[Webhooks] "demilestoned" payload does not include removed milestone details

### DISCUSSION THREAD

**Discussion by dgw** [community/community#24842] [Apps, API and Webhooks]
<p>Receiving a “demilestoned” event is less useful than it could be, because it just includes a “null” milestone object. Since the GitHub UI itself shows that “&lt;login&gt; removed this from the &lt;milestone name&gt; milestone”, the removed milestone’s information should be available to webhook receivers as well.</p>
<p>I’m not really sure what else to say, here. I expected to find an object in the payload somewhere, but it doesn’t seem to exist (unless I’m suffering from Wee Hours Coding Blindness again).</p>

**Comment by AndreaGriffiths11** [comment]
<p>Hi <a class="mention" href="/dgw">@dgw</a>,</p>
<p>Thanks for being here! A couple of questions to better understand how to help:</p>
<ul>
<li>Are you on an enterprise instance?</li>
<li>Were you authenticated?</li>
<li>Are you seeing a failure message?</li>
</ul>
<p>Apologies for the extra questions, I think I might be having lack of coffee issues :slight_smile:</p>

**Comment by dgw** [comment]
<p>This is on <a href="http://GitHub.com" rel="nofollow noopener">GitHub.com</a>. I don’t think there’s a way to authenticate webhook receivers… The whole webhook system is pretty fire-and-forget.</p>
<p>I did see an error on the webhook processing side because it was not programmed to handle a missing milestone object, but not on GitHub’s side.</p>

**Comment by AndreaGriffiths11** [comment]
<p>Thanks for the follow-up, I’ll pass this along to our ecosystem team and post any news back here.</p>

**Comment by dgw** [comment, marked as answer]
<p>I just happened to examine a “demilestoned” event payload from the history today, and noticed that there is a “milestone” object beside the “issue” now. It appears to contain the details of the removed milestone.</p>
<p>This might be solved.</p>
